### PR TITLE
Use warningf instead of warnf

### PR DESCRIPTION
**Pull request by ikitiki**
make it more homogeneous

### CODE REVIEW

**Comment by ikitiki** [comment]
👍

File: pkg/cluster/pg.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func (c *Cluster) closeDbConn() (err error) {
 		}
 		c.pgDb = nil
 	}
-	c.logger.Warning("attempted to close an empty db connection object")
+	c.logger.Warn("attempted to close an empty db connection object")
```
Comment (alexeyklyukin) [inline pkg/cluster/pg.go]: Shouldn't we stick with Warning in addition to Warningf?

Comment (ikitiki) [reply pkg/cluster/pg.go]: yeah, missed that

**Comment by alexeyklyukin** [comment]
👍

**Comment by ikitiki** [comment]
👍

**Pull request merged**